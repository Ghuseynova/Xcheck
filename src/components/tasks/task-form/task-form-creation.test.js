
import React from 'react';
import TaskFormCreation from './task-form-creation';
import { Provider } from 'react-redux';
import configureStore from 'redux-mock-store';
import { mount } from '../../../../jest.setup';

const mockStore = configureStore();


describe('Render TaskFormCreation', () => {

	it('render correctly', () => {
		const mockDispatchFn = jest.fn();
		const wrapper = mount(
			<Provider store={mockStore()}>
				<TaskFormCreation dispatch={mockDispatchFn} />
			</Provider>,
		);
		expect(wrapper).toBeDefined();
		expect(wrapper).toMatchSnapshot();
	});

});