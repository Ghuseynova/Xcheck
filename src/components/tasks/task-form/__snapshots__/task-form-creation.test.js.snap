// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `ShallowWrapper {}`;

exports[`Render TaskFormCreation render correctly 1`] = `ReactWrapper {}`;

exports[`check field form values r 1`] = `ShallowWrapper {}`;

exports[`check field form values renders correctly 1`] = `ShallowWrapper {}`;

exports[`render Renders correctly 1`] = `ShallowWrapper {}`;

exports[`render TaskFormCreation with initial values 1`] = `ShallowWrapper {}`;

exports[`render renders correctly 1`] = `ShallowWrapper {}`;

exports[`renders correctly 1`] = `ShallowWrapper {}`;
